ci/gpu/build.sh

```diff
@@ -113,4 +113,6 @@ else
     echo -e "\nTOP 20 SLOWEST TESTS:\n"
     # Wrap in echo to prevent non-zero exit since this command is non-essential
     echo "$(${WORKSPACE}/ci/getGTestTimes.sh testoutput.txt | head -20)"
+
+    ${WORKSPACE}/ci/gpu/test-notebooks.sh
```

Comment:
```suggestion
    ${WORKSPACE}/ci/gpu/test-notebooks.sh 2>&1 | tee nbtest.log
    EXITCODE=$? 
    python /${WORKSPACE}/ci/utils/nbtestlog2junitxml.py nbtest.log
```

As discussed in Slack to add the results JUnit to Jenkins, that way its easier for devs to parse out the failures in the build.